charter.md

```diff
@@ -1,19 +1,21 @@
 # Federated Identity Community Group Charter
 
-- [This Charter](https://fedidcg.github.io/charters/2021-08-02) (version 2021-08-02)
-- Previous Charter: n/a
-- Start Date: August 2, 2021
+- [This Charter](https://fedidcg.github.io/charters/2021-08-02) (version 2022-10-xx)
+- Previous Charter: (version 2021-08-02)
+- Community Group Start Date: August 2, 2021
 
 ## Goals
 
 The purpose of the Federated Identity Community Group is to provide a forum focused on incubating web features that will both support federated identity and prevent untransparent, uncontrollable tracking of users across the web. While the community group will take privacy concerns into consideration, these concerns will be balanced against the need to explore innovative ideas around federated authentication on the web.
 
-The group has both short- and long-term goals; the initial work focuses on the short-term goals as prioritized by the urgency of changes already underway, such as the phasing out of third-party cookies. This charter will be reviewed no later than one year from approval to potentially reset the goals and priorities.
+The group has both short- and long-term goals; the initial work focused on the short-term goals as prioritized by the urgency of changes already underway, such as the phasing out of third-party cookies. As the deprecation of third-party cookies moves ahead, the group's focus will expand to consider changes to link decoration and bounce tracking. This charter will be reviewed annually to potentially reset the goals and priorities.
```

Comment:
I'd maybe suggest something that emphasizes that we are not decreasing our focus on the deprecation of third party cookies, but rather incrementally allowing us to look a bit further in addition to.

I don't feel strongly about the words here, but here is one option that is more explicit:

```suggestion
The group has both short- and long-term goals; the initial work focused on the short-term goals as prioritized by the urgency of changes already underway, such as the phasing out of third-party cookies. That sense of urgency will continue and expand (e.g. via the extension of existing proposals or the creation of new proposals), but, as the deprecation of third-party cookies moves ahead, the group's attention will also start considering changes to link decoration and bounce tracking (since it has much bigger implications). This charter will be reviewed annually to potentially reset the goals and priorities.
```

Reply:
A few comments on Heather's original phrasing.

First, we're not deprecating third-party cookies, we're blocking or partitioning them and putting the unpartitioned ones behind the Storage Access API. Second, Safari and Firefox are already blocking unpartitioned third-party cookie access by default so "moving ahead" is not an accurate description of the browser landscape. Finally, I think it's better to use "navigational tracking" than link decoration and bounce tracking.

Thanks!

Reply:
+1.